src/functions/utils.php

```diff
@@ -123,3 +123,27 @@ function tribe_exit( $status = '' ) {
 		return call_user_func( $handler, $status );
 	}
 }
+
+if ( ! function_exists( 'tribe_get_request_var' ) ) {
+	/**
+	 * Tests to see if the requested variable is set either as a post field or as a URL
+	 * param and returns the value if so.
+	 *
+	 * Post data takes priority over fields passed in the URL query. If the field is not
+	 * set then $default (null unless a different value is specified) will be returned.
+	 *
+	 * @param string $var
+	 * @param mixed  $default
+	 *
+	 * @return mixed
+	 */
+	function tribe_get_request_var( $var, $default = null ) {
+		if ( isset( $_POST[ $var ] ) ) {
```

Comment:
Actually, realizing I totally misunderstood how Utils__Array worked: yep we could use that, but it's not it develop yet (I can add a note to update once it is, though).